manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -5873,28 +5880,34 @@ and update the appendix. -->
                       </para>
                     </note>
                   </para>
-
+<!-- Translators: You can add a sentence about your secondary regional exchanges from table
+"gnc-tbl-fq-yahoo-exchange-codes"in tips_appendix.xml -->
                   <para>Beachten Sie allerdings, dass Sie bei Verwendung von <quote>Yahoo as JSON</quote> zusätzlich das
-                    Börsenplatzkürzel des Wertpapiers anfügen sollten, z.B. <quote>PA</quote>
-                    für Paris, <quote>BE</quote> für Berlin usw. Beispiel: 12150.PA (ein
-                    Peugeot-Papier am Pariser Markt). 
+                    Börsenplatzkürzel des Wertpapiers anfügen sollten, z.B. <quote>F</quote> für Frankfurt,
+                    <quote>SW</quote> für Swiss oder <quote>VI</quote> für Vienna, usw. Als Beispiele sind
+                    hier VOW.F, NESN.SW und VOE.VI genannt. <quote>DE</quote> steht hier für
+                    <emphasis>Deutsche Börse XETRA</emphasis>.
                     <xref linkend="gnc-tbl-fq-yahoo-exchange-codes"></xref> enthält eine Liste von
                     Suffixen für verschiedene Märkte auf der ganzen Welt.
+<!-- Translators: Some fund companies create their own pseudo-symbols that differ from the regional codes, such as
+the TIAA-CREF fund as a regional fund. You are free to add corresponding tables from your language area in the
+tips_appendix.xml and to adapt the text here. -->
+                    <footnote>
+                      <para>Pseudo-Symbole für den amerikanischen Lehrer-Pensionsfond TIAA-CREF-Fonds sind in der Tabelle
+                        <xref linkend="PsSymbs-TIAA-CREF"></xref> aufgeführt.
+                      </para>
+                    </footnote>
                     <tip>
                       <para>Da einige Symbole eine spezielle Schreibweise erfordern, schauen Sie sich auch die Beispiele unter
                         <ulink url="&url-wiki;Online_Quotes#Setting_up_online_quotes_in_GnuCash" /> an.
```

Comment:
Eigentlich sollen am Ende die Links in die andere Richtung verweisen.

Der gelinke Abschnitt sollte sich doch in <q>Einrichtung der Kursaktualisierung</q> als Symbol-/Fehlersuche o.ä. integrieren lassen.

Genaugenommen geht es in dem Beispiel um Sonderzeichen des Befehlszeilen-Interpreter und wie man die quotet, damit sie als Teil des Symbols interpretiert werden, also Grundkurs Bash unter Linux.& dient da zur Umleitung der Ausgabeströme Output und Error.

Reply:
@fellen: Im Kapitel F::Q kann ich mir gut einen eigenen Abschnitt `Little Helpers` mit den ganzen Hilfsprogrammen vorstellen und den kompletten Bereich aus dem Wiki [Helper Scripts](https://wiki.gnucash.org/wiki/Online_Quotes#Helper_Scripts) übernehmen.

Reply:
Ja, würde gut als Symbolsuche o.ä. zwischen
[Installieren von Finance::Quote](xref:finance-quote-install)
und 
[Automatisierte Abfrage von Kursen](xref:finance-quote-scheduler)
passen.